frontend/src/RecordingPageComp/RecordingComp.jsx

```diff
@@ -70,18 +70,24 @@ function RecordingComp() {
     audio.src = audioSrc;
   }
 
+  useEffect(() => {
+    if (recorderControls.recordingTime === 10) recorderControls.stopRecording();
+  }, [recorderControls.recordingTime, recorderControls.stopRecording]);
```

Comment:
I don't think the `stopRecording` dependency is needed. The `recordingTime` dependency should suffice to ensure the condition will get evaluated properly. But lmk if you think otherwise!